libs/blocks/card/cardUtils.js

```diff
@@ -35,8 +35,19 @@ export const addFooter = (links, container, merch) => {
     if (parentElement && document.body.contains(parentElement)) parentElement.remove();
   });
 };
-
 export const addWrapper = (el, section, cardType) => {
+  const observeClassListChanges = (classToRemove) => {
+    const callback = (mutationsList) => {
+      for (const mutation of mutationsList) {
+        const { classList } = section;
+        if (mutation.type === 'attributes' && mutation.attributeName === 'class' && classList.contains(classToRemove)) classList.remove(classToRemove);
```

Comment:
`mutation.type === 'attributes'` is implied in `attributes: true` below and might be redundant

mutation.attributeName === 'class'` condition is implied in `attributeFilter: ['class']` and might be redundant

`classList.remove(classToRemove)` does nothing if CSS class being removed is not present in the list, `classList.contains(classToRemove)` condition can be omitted

Reply:
classList.contains(classToRemove) condition is added not to have an infinite loop.